midi_app_controller/gui/binds_editor.py

```diff
@@ -97,9 +140,31 @@ def _create_button_layout(self, button_id: int, button_name: str) -> QHBoxLayout
         action_combo = ActionsQComboBox(self.actions, action, self)
         self.button_combos.append((button_id, action_combo))
 
+        # Button label
+        button_label = QLabel(button_name)
+
+        # Button for lighting up the controller element
+        controller_disconnected = self.connected_controller is None
+        light_up_button = QPushButton("Light up")
+        light_up_button.setToolTip(f"Lights up the '{button_name}'")
+        light_up_button.setCursor(Qt.PointingHandCursor)
+        light_up_button.clicked.connect(lambda: self._light_up_button(button_id))
+
+        sizes = [2, 2, 6, 10]
+        elems = [
```

Comment:
maybe do `elems = [ (2, button_label), ...]` and later `for elem, size in elems`